index.js

```diff
@@ -483,18 +483,29 @@
 
   //. ### Classify
 
-  //# type :: Any -> String
+  //# type :: Any -> { namespace :: Maybe String, name :: String, version :: Integer }
```

Comment:
To avoid the awkwardness of built-in types forever being stuck at version zero, we could use `Maybe PositiveInteger` rather than `Integer`. I don't like the idea of an explicit `@0` being represented as `Nothing`, though.